actions/test

```diff
@@ -8,6 +8,7 @@ export PATH="$PATH:/snap/bin"
 FOCUS=$(action-get focus)
 SKIP=$(action-get skip)
 PARALLELISM=$(action-get parallelism)
+EXTRA_ARGS=$(action-get extra)
```

Comment:
you'll need to update the [actions.yaml](https://github.com/charmed-kubernetes/charm-kubernetes-e2e/blob/main/actions.yaml) to have this parameter available to the action